### DISCUSSION TITLE
What happened to download a stats?

### DISCUSSION THREAD

**Discussion by theDevAyan** [pystardust/ani-cli#611] [Q&A] [answered]
So i have been using ani-cli to download anime and earlier when downloading it showed the amount downloaded out of the the total size. But now it shows the length downloaded and frames and other stuff printing it every few seconds and completely filling my terminal.

Is there any way to restore it like it was. And i would suggest that it would be nice if there was a simple clean status bar.

**Comment by CoolnsX** [comment, marked as answer]
There r two downloader used now.. one for downloading .MP4 files and one for .m3u8 file..
Earlier downloader u saw is aria2.. which supports .MP4 and parallel downloading
Another one that shows frame is ffmpeg... Which supports downloading .m3u8 but doesn't support parallel downloading..

And lately gogoanime is giving more and more .m4u8 files.. so we can't do anything about it..

**Reply by theDevAyan** [reply]
but if its downloading from ffmpeg which downloads .m3u8 then why when i check the properties of the downloaded video it shows as .mp4 format

**Comment by CoolnsX** [comment]
Ffmpeg convert it to MP4 while downloading.. ig u need to learn more about ffmpeg.. it's a great tool for processing video files...

**Reply by theDevAyan** [reply]
Thanks man for the help

**Reply by CoolnsX** [reply]
No problem
